posts/2021-08-30-open-liberty-stack-gradle.adoc

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "Cloud-native Development of Gradle Built Applications with the Open Liberty Devfile Stack"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mezarin
+author_github: https://github.com/mezarin
+seo-title: Cloud-native Development of Gradle Built Applications with the Open Liberty Devfile Stack - OpenLiberty.io
+seo-description: Get true-to-production development of Gradle built applications directly in a Kubernetes cluster
+blog_description: "Get true-to-production development of Gradle built applications directly in a Kubernetes cluster"
+---
+= Cloud-native Development of Gradle Built Applications with the Open Liberty Devfile Stack
+Edward Mezarina <https://github.com/mezarin>
+
+The Open Liberty devfile stack now allows you build and deploy new and existing cloud-native applications using Gradle. The stack leverages link:https://github.com/OpenLiberty/ci.gradle[Open Liberty Gradle Plugin] capabilities to manage the Liberty server and Gradle built applications.
+
```

Comment:
would be helpful to add a sentence or two as an overview of what the post will cover. Something like, "In this post, we'll use the Open Liberty devfile stack and Openshift do to deploy a sample application to a Kubernetes cluster"